### DISCUSSION TITLE
is it possible to separate two operators fields and put it between two value fields?

### DISCUSSION THREAD

**Discussion by kellywang1234** [react-querybuilder/react-querybuilder#660] [General]
Hi team, I have a question that needs your help and input!
I have two operators and two value fields which are all grouped together. I'd like to separate each operator and put it between two value fields. Here is the code example https://codesandbox.io/p/sandbox/react-query-builder-two-value-valueeditor-forked-dw4tzf?file=%2Fsrc%2FApp.tsx
So the outcome could be 
"Two values, =, Value1, = Value2" Thank you in advance!

**Comment by jakeboone02** [comment]
With the need to track two operators and two values, I would probably recommend ignoring the actual `operator` property of the rule altogether. Leave it as "=" or something then track both operators and both values in the `value` property.

You've already implemented a mechanism for tracking two values in one `value` property, so you can extend that to include each corresponding operator.

**Comment by kellywang1234** [comment]
thank you!
